sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakeFileClient.java

```diff
@@ -1221,7 +1221,7 @@ public Response<PathProperties> readToFileWithResponse(String filePath, FileRang
                     .setRequestConditions(Transforms.toBlobRequestConditions(requestConditions))
                     .setRetrieveContentRangeMd5(rangeGetContentMd5).setOpenOptions(openOptions), timeout,
                 context);
-            return new SimpleResponse<>(response, Transforms.toPathProperties(response.getValue()));
+            return new SimpleResponse<>(response, Transforms.toPathProperties(response.getValue(), Transforms.getEncryptionContext(response)));
```

Comment:
`Transforms.getEncryptionContext(response)` always returns null since encryption context is not settable on blob client, should we keep as is or return null here for encryptionContext? @seanmcc-msft @alzimmermsft

Reply:
You can get the encryption context from the headers on the raw response.  We should be returning encryption context here, as we do in .NET.

Reply:
Checked responses that we get from `readToFileWithResponse()` in Fiddler - `x-ms-encryption-context` is not present in the response headers. It is present in the response headers for `readWithResponse()` method though.